public/app/features/alerting/unified/components/expressions/Expression.tsx

```diff
@@ -402,7 +402,7 @@ function FrameRow({ frame, index, isAlertCondition, isRecordingRule }: FrameProp
   const styles = useStyles2(getStyles);
 
   const name = getSeriesName(frame) || 'Series ' + index;
-  const value = getSeriesValue(frame);
+  const value = frame.fields.at(0)?.values.at(0);
```

Comment:
I think we should use `getSeriesValue` here , right?

Reply:
Yes! good catch